tests/clickhouse-test

```diff
@@ -789,6 +791,10 @@ def get_localzone():
     return os.getenv("TZ", "/".join(os.readlink("/etc/localtime").split("/")[-2:]))
 
 
+def _running_on_arm():
```

Comment:
We might also check it queering server and checking build options, as we do for test tags